src/component/PollItem.js

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
+import { FaTrashAlt, FaEdit } from "react-icons/fa";
+import { IoBarChart } from "react-icons/io5";
+import { ADMIN_ID } from "../utils/constantData";
+import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { votedPollOption } from "../redux/reducer/pollListReducer";
+
+const PollItem = ({ poll, viewPollVoteChart, showDeleteModal }) => {
+  const [userData, setUserData] = useState({});
+  const [selectedOption, setSelectedOption] = useState(null);
+  const [voted, setVoted] = useState(
+    JSON.parse(localStorage.getItem(`poll_${poll.title}`)) || false
```

Comment:
use a selector for this too, as the mechanism you are using, is also wrong, because the poll.title is not unique, multiple pill titles can be the same, use array or use some unique key for localstorage @amitark7